src/main/java/frc/robot/subsystems/usablename/UsableName.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.usablename;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.subsystems.Xavier;
+
+public class UsableName extends SubsystemBase implements UsableNameIO
+{
+	private final Xavier xavier;
+	public UsableNameIODataAutoLogged data;
+
+	public UsableName(Xavier xavier)
```

Comment:
Why do you have xavier as class referenced in Usable Name.

Reply:
The three classes should be UsableName, UsableNameIO, and UsableNameIOXavier

Reply:
UsableName should be the only one to extend Subsystsem

Reply:
UsableName should have UsableNameIOInputsAutoLogged and UsableIO. It should not know that UsableIO instance is in fact UsableIOXavier